lib/wickedbase.pm

```diff
@@ -562,18 +557,28 @@ sub post_run {
     $self->{wicked_post_run} = 1;
 
     $self->do_barrier('post_run');
+    if ($self->{name} ne 'before_test' && get_var('WICKED_TCPDUMP')) {
+        script_run('kill ' . get_var('WICKED_TCPDUMP_PID'));
```

Comment:
This shouldn't fail, would you like to ignore a crashed process? ;)

Reply:
yes I do , this test is not testing tcpdump , status of the test should not be influenced by tcpdump.